build.gradle

```diff
@@ -32,6 +32,10 @@ shadowJar {
     }
 }
 
+tasks.compileJava {
+    options.compilerArgs += '-Xlint:deprecation'
+}
+
 spotless {
     java {
```

Comment:
In a separate PR, can you turn on spotless formatting of gradle files?

Reply:
I am sorry I did not get it, do you mean apply spotless formatting to gradle files when compileJava it?

Reply:
No, I meant adding a gradle formatting block to spotless, similar to this: https://github.com/eisop/checker-framework/blob/388fdb094942de42e871a96171fadb80c4d1a0e1/build.gradle#L309

Reply:
Done! Please see #220 :)